sympy/vector/tests/test_printing.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
This line is still needed if there are non-ASCII characters in the file because there are tests that only allow non-ASCII characters in certain files.

Reply:
Thank you for pointing it out - I've added it again